src/main/java/edu/ucla/library/bucketeer/utils/UCLAFilePathPrefix.java

```diff
@@ -4,36 +4,54 @@
 import java.io.File;
 import java.nio.file.Paths;
 
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 /**
  * A prefix for UCLA image masters. The prefix paths are defined in the <a href=
  * "https://github.com/UCLALibrary/californica/blob/master/app/importers/californica_mapper.rb#L180">Californica
  * code</a>.
  */
 public class UCLAFilePathPrefix implements IFilePathPrefix {
 
+    /**
+     * The <code>serialVersionUID</code> of UCLAFilePathPrefix.
+     */
+    private static final long serialVersionUID = -7564099601987867700L;
+
     private static final String MASTERS_DIR_PATH = "Masters/";
 
     private static final String DL_MASTERS_DIR_PATH = "Masters/dlmasters/";
 
-    private final String myRootDir;
+    @JsonProperty("uclaRootDir")
+    private String myUCLARootDir;
+
+    /**
+     * Creates a prefix for UCLA's image sources.
+     */
+    @SuppressWarnings("unused")
```

Comment:
are we sure we want to leave this here?

Reply:
I'm assuming this refers to the SuppressWarnings annotation and not the private constructor(?) Is there a reason we wouldn't? Eclipse warns about it being an unused constructor because it's not called from within the class itself (where one might expect a private constructor to be called). It's there because Jackson needs an empty constructor to instantiate the object when going from JSON to Java, so the warning is a false warning in this case -- yes, it's not used by the class, but it is used.

Reply:
okeydoke